client/imports/app/modules/core/accounts/accounts.ts

```diff
@@ -56,17 +56,17 @@ export class AccountComponent implements OnInit, OnDestroy {
 	ngOnInit() {
 		//**** time limit check condition if it excced 1 hrs
 		// if login time is more than 1 hr then we should logout the user.
-		if (localStorage.getItem("login_time")) {
-			var login_time = new Date(localStorage.getItem("login_time"));
+		if (this._local.getLocaldata("login_time")) {
+			var login_time = new Date(this._local.getLocaldata("login_time"));
```

Comment:
i think i had mentioned previously to put this code in a seperate function, i.e make a single code to check login time of user and use the function in all the controllers.